### PR TITLE
ZENKO-1002 improvement: remove MetalK8s instructions

### PR DESCRIPTION
**Pull request by rahulreddy**
** Please review second commit only **

### CODE REVIEW

File: docs/Installation/Installation_Guide.rst [hunk 1]
```diff
@@ -102,298 +79,6 @@ If you are behind a proxy, add the following lines to your local machine’s
     https_proxy=http://user;pass@<my-ip>:<my-port>
```
Comment (giacope) [inline docs/Installation/Installation_Guide.rst]: This proxy stuff can be deleted as well. No clue where it came from and nothing at the application level depends on it AFAIK.

**Review by tmacro — APPROVED** [review]

**Review by philipyoo — APPROVED** [review]

**Review by alexanderchan-scality — APPROVED** [review]

**Comment by rahulreddy** [comment]
@bert-e wait

**Comment by rahulreddy** [comment]
Let's wait for @wabernatScality , @smaffulli and @NicolasT to review this as it will undo a lot of work that @wabernatScality has put in.

**Comment by dora-korpar** [comment]
Are these instructions being moved to the metalk8s repo? Cos I agree that this probably isn't the place for them, but they're the best metalk8s instructions imo, so I'd prefer not to lose them

**Comment by JianqinWang** [comment]
Mis-clicked, apologies

**Review by giacope — DISMISSED** [review]
@rahulreddy There are still some references to MetalK8s in the "Get Ready" section.

**Comment by rahulreddy** [comment]
I know..let's wait for the stakeholders to chime-in before I spend anymore time on this.

**Comment by wabernat** [comment]
What Rahul and Dora said. Building these instructions also burned a lot of Salim's time herding me through the process.

Provided I can copy some of the non-redundant MetalK8s instructions over to the MetalK8s Quickstart without fomenting fresh controversies, I have zero objection to removing the MetalK8s instructions from Zenko. It makes good sense to do so. 

The MetalK8s steps I added to the beginning of the Zenko installation were in specific response to a prominent customer's predicted demands, and my comically inept attempts to follow the Quickstart instructions and attendant regroupings. The MetalK8s Quickstart in places (at least at the time I looked at it last month) assumed a level of sophistication that I lack, and we should assume likewise wanting in some of our customers.

**Review by wabernat — APPROVED** [review]
I have moved the deleted sections to the MetalK8s Quickstart and pushed them. I have no issues with what remains here

**Comment by wabernat** [comment]
Giacomo: I'm going to dismiss your comment concerning MetalK8S references remaiing in the doc, owing to present urgency in getting us all in sync. This one needs to go through before my others.

**Comment by wabernat** [comment]
Closing. 

The purpose of this PR was to remove MetalK8s instructions from the Installation Guide. That has happened in the development/1.0 branch. This is breaking on RST parser errors. Fixing them will take a long time and accomplish nothing.

**Pull request closed without merging**